src/components/LaunchDetailsPage/components/LaunchCardContent/components/CardDetails/CardDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Typography } from '@mui/material';
+import { launchDetails } from '../../../../mocks/launchDetails';
+import './CardDetails.scss';
+
+export const CardDetails = () => {
+  const detailsCaptions: string[] = ['Area:', 'Status:', 'Date:'];
+  const detailsValues: string[] = [
+    `${launchDetails.pad.name}, ${launchDetails.pad.location.name}`,
+    launchDetails.status.name,
+    launchDetails.net,
+  ];
+
+  return (
+    <div className="card-details">
+      {detailsCaptions.map((caption: string, index: number) => (
+        <Typography variant="body1" key={index}>
+          {caption}
+        </Typography>
+      ))}
+      {detailsValues.map((value: string, index: number) => (
+        <Typography variant="body1" key={index}>
+          {value}
+        </Typography>
+      ))}
```

Comment:
creat function for rendering above and use here 2 times

Reply:
done